src/ripple/core/impl/Config.cpp

```diff
@@ -330,13 +378,13 @@ void Config::loadFromString (std::string const& fileContents)
             NODE_SIZE = 4;
         else
         {
-            NODE_SIZE = beast::lexicalCastThrow <int> (strTemp);
+            NODE_SIZE = beast::lexicalCastThrow <std::size_t>(strTemp);
 
-            if (NODE_SIZE < 0)
-                NODE_SIZE = 0;
-            else if (NODE_SIZE > 4)
+            if (NODE_SIZE > 4)
                 NODE_SIZE = 4;
         }
+
+        assert (NODE_SIZE <= 4);
```

Comment:
I don't see any way for this assert to fire...am I missing something?

Reply:
Not anymore. I'll remove it.